### ISSUE TITLE
[Screen Reader - LS command]: Narrator stays silent and does not announce any information after entering LS command.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8463]
**User Experience:**
Users who rely on screen reader would be impacted and not able to get the information as narrator stays silent and does not announce any information after entering LS command.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0

**Repro Steps:**
1. Open Windows terminal app.
2. Launch Narrator.
3. Type "LS" and press enter
4. Directory list gets open.
5. Observe the issue.

**Actual Result:**
Narrator stays silent does not announce any information after entering LS command.

**Expected Result:**
Narrator should announce the proper information about the directory list getting open after entering LS Command.

**Note:** Issue does not repro with NVDA. NVDA announces the information after entering LS Command.

**Guideline Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BF98F4235-B6BD-47D3-8DA4-F28B9B6FBFA8%7D&file=Guideline%20101%20%E2%80%93%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties.docx&action=default&mobileredirect=true&cid=9ec86a2d-7fab-4e44-810c-950ae86b7a69

[Narrator_Usable_LS Comman_No info by narrator.zip](https://github.com/microsoft/terminal/files/5629546/Narrator_Usable_LS.Comman_No.info.by.narrator.zip)

**Comment by zadjii-msft** [comment]
@carlos-zamora You wanna take a look at this? This _sounds_ like a pretty major Narrator regression, though I'm not familiar enough with the scenario to be sure.

**Comment by carlos-zamora** [comment]
CC @DHowett 
Unfortunately, this is a known issue. IIRC Narrator in particular needs to update their approach to rely on UIA Events we dispatch. NVDA does that and works fine (though I don't know what JAWS does). We need to coordinate with Narrator so that they can commit a fix.

**Comment by v-rpundir** [comment]
We are tracking this bug here in Narrator VSO https://microsoft.visualstudio.com/OS/_workitems/edit/36561781

**Comment by v-rpundir** [comment]
Bug Update: Issue also repro with JAWS.

**Comment by carlos-zamora** [comment]
(FYI for anybody who comes along this thread) Narrator and JAWS are both aware of this issue and are working to make changes on their end to fix this.

**Comment by carlos-zamora** [comment]
@v-rpundir this should be fixed now as of v1.13

**Comment by v-rpundir** [comment]
Closing as fixed on v1.13 with Narrator (22616) and JAWS 2021.
